proposals/description/tip_118.ts

```diff
@@ -379,25 +358,44 @@ const tip_118: TemplatedProposalDescription = {
       description: 'Move all agEUR to redeemer'
     },
     {
-      target: 'collateralizationOracle',
+      target: 'angleEuroRedeemer',
       values: '0',
-      method: 'removeDeposits(address[])',
-      arguments: (addresses) => [[addresses.agEurUniswapPCVDeposit, addresses.uniswapLensAgEurUniswapGauge]],
-      description: 'Remove agEUR addresses from CR oracle'
+      method: 'redeemAgEurToDai()',
+      arguments: (addresses) => [],
+      description: 'Redeem all agEUR for DAI and send to DAI PSM'
     },
+
+    // 4. Unset various safe addresses - deprecated PSMs and agEurUniswapPCVDeposit
     {
       target: 'pcvGuardianNew',
       values: '0',
-      method: 'unsetSafeAddress(address)',
-      arguments: (addresses) => [addresses.agEurUniswapPCVDeposit],
-      description: 'Remove agEurUniswapPCVDeposit from safe addresses'
+      method: 'unsetSafeAddresses(address[])',
+      arguments: (addresses) => [
+        [addresses.ethPSM, addresses.lusdPSM, addresses.raiPriceBoundPSM, addresses.agEurUniswapPCVDeposit]
```

Comment:
Should we remove [aaveEthPCVDeposit](https://etherscan.io/address/0x5B86887e171bAE0C2C826e87E34Df8D558C079B9) from safe addresses ? it's empty and is getting removed from CR oracle.

Should we remove [turboFusePCVDeposit](https://etherscan.io/address/0x2c47Fef515d2C70F2427706999E158533F7cF090) from safe addresses ? it's empty.

[raiPriceBoundPSM](https://etherscan.io/address/0x5ddE9B4b14eDf59CB23c1d4579B279846998205e) is not a safe address anymore since 2022-06-28, can remove it from the array